app/src/test/java/org/simple/clinic/patient/PatientMocker.kt

```diff
@@ -335,8 +324,8 @@ object PatientMocker {
         phoneNumber = phoneNumber,
         appointment = appointment,
         bloodPressure = bloodPressureMeasurement,
-        riskLevelIndex = calculatedRiskLevel
-    )
+        isAtHighRisk = isHighRisk
+        )
```

Comment:
Small formatting issue, it should be automatically resolved by the code formatting action